configurer/linux/enterpriselinux/amazon.go

```diff
@@ -13,6 +14,12 @@ type AmazonLinux struct {
 	configurer.Linux
 }
 
+func (a AmazonLinux) Hostname(h os.Host) string {
```

Comment:
I've used `l` (downcase `L`) as the variable name everywhere else.

```suggestion
func (l AmazonLinux) Hostname(h os.Host) string {
```